aicsimageio/readers/default_reader.py

```diff
@@ -362,7 +362,7 @@ def _read_delayed(self) -> xr.DataArray:
                     image_data,
                     dims=dims,
                     coords=coords,
-                    attrs=metadata,
+                    attrs={constants.METADATA_UNPROCESSED: metadata},
```

Comment:
this unprocessed/processed thing needs to be documented somewhere (I'm going in order so may not have reached docs yet)

Reply:
It isn't documented! But will!